app/controllers/index.rb

```diff
@@ -0,0 +1,5 @@
+require 'sinatra'
+
+get '/' do
+  p "Stephanie"
```

Comment:
:panda_face: Why are you using p here? What does a get request return in the response body?